students/sangwoong/users/views.py

```diff
@@ -56,14 +58,20 @@ def post(self, request):
             data          = json.loads(request.body)
             user_account  = data["account"]
             user_password = data["password"]
+            user_id       = User.objects.get(account = user_account).id
 
             if not User.objects.filter(account = user_account).exists() :
                 return JsonResponse({"Message": "INVALIDE_USER"}, status=401)
 
-            if User.objects.get(account = user_account).password != user_password :
-                return JsonResponse({"Message": "INVALID_USER"}, status=401)
+            hashed_password = user_password.encode("utf-8")
+            saved_password  = User.objects.get(account = user_account).password
+            
+            if not bcrypt.checkpw(hashed_password, saved_password.encode("utf-8")) :
+                return JsonResponse({"Message" : "INVALID_USER"}, status=401)
+
+            access_token = jwt.encode({"user_id" : user_id}, SECRET_KEY, ALGORITHM)
 
-            return JsonResponse({"Message": "SUCCESS"}, status=200)
+            return JsonResponse({"ACCESS_TOKEN": access_token, "Message": "SUCCESS"}, status=200)
```

Comment:
JSON으로 반환 되는 데이터의 key 값은 일반적으로 소문자로만 구성합니다.(스네이크 케이스)
다른 return들도 수정할 부분이 있다면 찾아서 같이 수정해 주세요.
```suggestion
            return JsonResponse({"access_token": access_token, "message": "SUCCESS"}, status=200)
```